### PR TITLE
Bug 1169636 - [mozdevice] Swapping out log stream parser

### PR DESCRIPTION
**Pull request by eliperelman**

### CODE REVIEW

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=53fd0e687b5ea9af4688c2232a5305bbf0446e31

File: tools/mozdevice/util.js [hunk 1]
```diff
@@ -5,8 +5,30 @@ var unzip = require('unzip');
...
+  brief: XRegExp("^(?<level>[VDIWEAF])\\/(?<tag>[^)]{0,23}?)\\(\\s*(?<pid>\\d+)\\):\\s(?<message>.*)$"),
+  threadtime: XRegExp("^(?<timestamp>\\d\\d-\\d\\d\\s\\d\\d:\\d\\d:\\d\\d\\.\\d+)\\s*(?<pid>\\d+)\\s*(?<tid>\\d+)\\s(?<level>[VDIWEAF])\\s(?<tag>.*?):\\s(?<message>.*)$"),
+  time: XRegExp("^(?<timestamp>\\d\\d-\\d\\d\\s\\d\\d:\\d\\d:\\d\\d\\.\\d+):*\\s(?<level>[VDIWEAF])\\/(?<tag>.*?)\\((?<pid>\\s*\\d+)\\):\\s(?<message>.*)$"),
+  process: XRegExp("^(?<level>[VDIWEAF])\\(\\s*(?<pid>\\d+)\\)\\s(?<message>.*)$"),
+  tag: XRegExp("^(?<level>[VDIWEAF])\\/(?<tag>[^)]{0,23}?):\\s(?<message>.*)$"),
+  thread: XRegExp("^(?<level>[VDIWEAF])\\(\\s*(?<pid>\\d+):(?<tid>0x.*?)\\)\\s(?<message>.*)$"),
+  ddms_save: XRegExp("^(?<timestamp>\\d\\d-\\d\\d\\s\\d\\d:\\d\\d:\\d\\d\\.\\d+):*\\s(?<level>VERBOSE|DEBUG|ERROR|WARN|INFO|ASSERT)\\/(?<tag>.*?)\\((?<pid>\\s*\\d+)\\):\\s(?<message>.*)$")
+};
+
+var LEVELS = {
```
Comment (rwood-moz) [inline tools/mozdevice/util.js]: I don't see where the var / obj LEVELS is actually used

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=f3d510a4666acb2a7fb392833bc145a910b42681

File: tools/mozdevice/util.js (hunk 1 above)
Comment (eliperelman) [inline tools/mozdevice/util.js]: Good catch, I couldn't where it was used in the original source either. DELETED.

**Comment by autolander** [comment]
http://docs.taskcluster.net/tools/task-graph-inspector/#6EQxayjfRe6-Fju6fj9r_g

The pull request failed to pass integration tests. It could not be landed, please try again.

**Pull request merged**